pyccel/ast/mathext.py

```diff
@@ -82,9 +84,14 @@
 # Base classes
 #==============================================================================
 class MathFunctionBase(Function, PyccelAstNode):
+    """Abstract base class for the Math Functions"""
     _shape = ()
     _rank  = 0
 
+class PycMathFunctionBase(Function, PyccelAstNode):
+    """Abstract base class for the Math Functions implemented in Pyc_math
+    libray"""
```

Comment:
I'm happy to approve this PR once this class has been removed

Reply:
o n t h e w a y.